rfcs/text/0000-conceptual-data-model.md

```diff
@@ -0,0 +1,111 @@
+The goal of the experimental metadata is to capture a detailed, accurate representation of the experimental process and the scientific data within the DCP.  The experimental metadata is closely coupled to the experimental data and expected to evolve as experimental methods and consumer needs evolve.
+
+The full scientific data content of the DCP is represented by the experimental metadata model, including all versions of data and metadata instances.  Programmatic access to all versions of metadata supports the exploration and analysis of the provenance of HCA data.
+
+The addition of data and metadata update functionality (AUDR) has increased the complexity of the data model. A rigorous definition of the full data model is needed to communicate about and to interpret the versioned graph consistently and accurately.
+
+
+### User Stories
+
+As a DCP developer, to develop software to manage versioned data,
+I would like to understand how to interpret the data and communicate with other developers in a clear and precise manner.
+
+As a data operations engineer, to construct a data release, I would like to understand what specific data and metadata instances to include.
+
+As a bioinformatician, to replicate an experiment, I would like to understand the provenance of the relevant data files from a data release.
+
+## Detailed Design
+
+The experimental metadata is structured as DAG of instances, with basic categories of *Project*, *Biomaterial*, *Protocol*, *Process*, and *File*.  Each of these types may have more detailed derived types.  This structure is defined in more detail in [Overview of Metadata Structure](https://github.com/HumanCellAtlas/metadata-schema/blob/master/docs/structure.md).
+
+Conceptually, the full, versioned metadata graph consists of all metadata instance versions in the DCP.  However, there are multiple ways to traverse the graph based on the question being asked:
+
+
+* Head graph: The top-down traversal of the most current version of metadata instances.  This does not include obsolete instances.  This view of the graph is normally used for browser and selection of the latest results
+* Provenance graph: This is the reverse-traversal graph from a specific version of a metadata instance, visiting the exact versions of preceding metadata instances.  It records the context of an experiment when it was run.
+* Amended provenance graph: The provenance graph with corrections that are inconsequential, such as correcting natural language descriptions.  This graph is used to produce data releases.
+
+
+The head graph is the graph of the latest version of all the metadata, excluding those that are obsolete due to a restructuring update or those that are flagged as erroneous.  This is the view normally used in selectinzΩg data sets and exploring the atlas.  Traversing from the roots to result in a branching set of paths to the leaves.  Normally, this graph would be generated by traversing via UUID, ignored the version timestamps.
```

Comment:
s/selectinzΩg/selecting

Reply:
thanks @Bento007 .  how did I ever get "Ω" in the text ?!?!